src/client/src/system/openFin/openFin.js

```diff
@@ -57,4 +58,50 @@ export default class OpenFin {
         return disposables;
       });
   }
+
+  viewChartIQ(symbol): void {
+    this.getChartIQInstance(symbol, this.refreshChartIQ, this.startChartIQ);
+  }
+
+  getChartIQInstance(symbol, callback, errorHandler):fin.desktop.Application{
+    let chartIqAppId = 'ChartIQ';
+    fin.desktop.System.getAllApplications(function(apps) {
+      let chartIqApp = _.find(apps, ((app) => {
+        return app.isRunning && app.uuid === chartIqAppId;
+      }));
+      if(chartIqApp) {
+        if (callback) callback(symbol);
+      } else {
+        if (errorHandler) errorHandler(symbol);
+      }
+    });
+  }
+
+  refreshChartIQ(symbol){
+    let interval = 5;
+    let chartIqAppId = 'ChartIQ';
+    console.log(`Publishing message to chartIQ with symbol ${symbol} and interval ${interval}`);
+    fin.desktop.InterApplicationBus.publish('chartiq:main:change_symbol', { symbol: symbol, interval: interval });
+  }
+
+  startChartIQ(symbol){
+    let interval = 5;
+    let chartIqAppId = 'ChartIQ';
+    console.log('Chart IQ was not running, so starting new app with symbol = ' + symbol + ' and interval ' + interval);
```

Comment:
There is a lot of cleaning up that was done on a different branch, including deleting the unused code, string templates and logging. I'll remove the newly added console.log statements from this branch and will add it to the logger in the notifications branch